core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -3881,13 +3881,41 @@ public static List compact(List list) {
   }
 
   /** Support the ARRAY_DISTINCT function. */
-  public static List distinct(List list) {
+  public static List arrayDistinct(List list) {
```

Comment:
why are we going to rename public methods?
that could impact downstream dependencies